db/migrate/20150403173410_add_has_joined_to_cud.rb

```diff
@@ -0,0 +1,27 @@
+class AddHasJoinedToCud < ActiveRecord::Migration
+
+  def up
+
+  	add_column :courses, :public, :boolean
+    add_column :courses, :requires_permission, :boolean
+    add_column :courses, :website_url, :string
+
+    add_column :course_user_data, :has_joined, :boolean
+
+    CourseUserDatum.all.each do |cud|
```

Comment:
Use 'find_each' here.  Any reasons some lines in this file are 3-indented/not aligned?